OWNERS_ALIASES

```diff
@@ -4,16 +4,13 @@ aliases:
   - chaodaiG
   - chases2
   - cjwagner
-  - fejta
   - listx
   - michelle192837
   - mpherman2
 
   # https://github.com/kubernetes/sig-release/blob/master/release-managers.md#build-admins
   build-admins:
-  # TODO: add juanfescobar after https://github.com/kubernetes/org/issues/3193
-    - BenTheElder
-    - juanfescobar
+    - BenjaminKazemi
```

Comment:
xref https://github.com/kubernetes/org/issues/3600, Juan is no longer in build admins oncall, nor am I since I left engprod.